tests/bdd/xpaths.py

```diff
@@ -74,7 +75,7 @@ class addZvol:
 
 class advanced:
     title = '//h1[contains(.,"Advanced")]'
-    systemDatasetPool_card = '//h3[text()="System Dataset Pool"]'
+    systemDatasetPool_card = '//h3[text()="Storage"]'
```

Comment:
Is it possible there is a more robust selector than the text? Not sure how often the text will be changing if the future so it may not be an issue.

Reply:
We have an open discussion on the future setup with Dev right now. This is acceptable for now